deploy.rb

```diff
@@ -95,8 +95,9 @@ def comment_out_branch_prompt!
 deploys = {}
 repo_infos.each do |repo_info|
   repo = repo_info['repo']
+  branch_name = repo_info['branch_name'] || 'master' # default to master if not supplied in the yaml file
```

Comment:
this is fine;  you could also set the parameter default of update_repo to "master" but whatevs.